cli/cmd/install.go

```diff
@@ -475,6 +290,11 @@ func render(w io.Writer, values *l5dcharts.Values, stage string, valuesOverrides
 	}
 
 	_, err = w.Write(buf.Bytes())
+	if err == nil {
+		fmt.Fprintln(os.Stderr, "Rendering Linkerd CRDs...")
+		fmt.Fprintln(os.Stderr, "Next, run `linkerd install | kubectl apply -f -` to install the control plane.")
```

Comment:
`linkerd check --pre` ends up calling this function through `install`. We only care about the checks in that command, but we now hit these `Fprintln`s and it's unexpected output.

For example

```shell
$ linkerd check --pre
Rendering Linkerd CRDs...
Next, run `linkerd install | kubectl apply -f -` to install the control plane.

Linkerd core checks
===================

kubernetes-api
--------------
√ can initialize the client
...
```

We don't care about the rendered CRDs here and we shouldn't run that command next; we haven't actually ran `linkerd install --crds` yet.

I was curious why we need to even render the manifests as part of `linkerd check --pre` and it's used by the `can create non-namespaced resources` check—so we do need to hit this codepath, but it'd be nice if we didn't print these as part of that.